target/linux/ramips/image/common-sercomm.mk

```diff
@@ -125,8 +135,17 @@ define Device/sercomm_cxx
 	append-ubi | sercomm-factory-cqr | sercomm-mkhash
 endef
 
-define Device/sercomm_dxx
+define Device/sercomm_cxx_dxx
```

Comment:
Please move this block up before the `Device/sercomm_cxx` which makes use of it. Then it is easier to read.

Reply:
Done.